tests/migration/online_migration/yast2_migration.pm

```diff
@@ -220,10 +220,13 @@ sub run {
             send_key 'alt-t';
         }
     }
-    assert_screen ['yast2-migration-installupdate', 'yast2-migration-proposal'], 700;
-    if (match_has_tag 'yast2-migration-installupdate') {
+    if (check_screen('yast2-migration-installupdate', 700)) {
```

Comment:
when yast2-migration-proposal can be match, in this logic seems SKIP the match action.

Reply:
The license agreement is shown between yast2-migration-installupdate and yast2-migration-proposal.  While from the test results, the yast2-migration-proposal and the yast2-migration-summary are in the same page and match different needle, and the cases match the needle of yast2-migration-summary. So I remove the yast2-migration-propoasl here.